app/models/manageiq/providers/autosde/storage_manager/ems_refresh_workflow.rb

```diff
@@ -0,0 +1,47 @@
+class ManageIQ::Providers::Autosde::StorageManager::EmsRefreshWorkflow < ManageIQ::Providers::EmsRefreshWorkflow
+
```

Comment:
re: https://github.com/ManageIQ/manageiq-providers-autosde/pull/172/files#r963691043

```suggestion
  class << self
    def start_refresh(target_class, target_id, ems_id, task_id, options = {})
      options.reverse_merge!(:target_class => target_class, :target_id => target_id, :ems_id => ems_id, :native_task_id => task_id)
      ManageIQ::Providers::Autosde::StorageManager::EmsRefreshWorkflow.create_job(options).tap { |job| job.signal(:start) }
    end
  end
```

This should allow you to call:
```
ext_management_system.class::EmsRefreshWorkflow.start_refresh(self.class.name, id, ext_management_system.id, task_id)
```